satellite/console/consoleweb/consoleql/consoleerrors.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package consoleql
+
+import (
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/satellite/console"
+)
+
+// Error messages
+const (
+	internalErrDetailedMsg = "It looks like we had a problem on our end. Please try again"
+)
+
+// errConsoleInternalDetailed describes detailed error message for internal error
+var errConsoleInternalDetailed = errs.New(internalErrDetailedMsg)
+
+// HandleError returns detailed error if such error handles
+func HandleError(err error) error {
+	switch {
+	case console.ErrConsoleInternal.Has(err):
+		return errConsoleInternalDetailed
```

Comment:
Use `errs.New("It looks like we had a problem on our end. Please try again")` here, rather than the separate variables. Otherwise the stack that's associated with the error is wrong.

There's probably also a way to still keep the original error attached to the error, such that it doesn't get exposed to the public.

I wonder whether we should have a way to hide errors at the global level, such that you don't have to write `HandleError` everywhere.

Reply:
We plan to move all console errors to separate typed errors. This is first part of implementation which will be extended and apparently changed.

Reply:
Also, I don't completely understand about error's stack. We don't wrap error, we just return user friendly message on console front-end part